### ISSUE TITLE
Protocols for finetuning on CIFAR10 or CIFAR100.

### ISSUE THREAD

**Issue by alirezaabdollahpour** [mlfoundations/wise-ft#17]
Hi @gabrielilharco  ;)
Could you kindly provide the protocols you use for finetuning on CIFAR10 or CIFAR100?
For example, factors such as the learning rate and the number of epochs... Please refer to Figure 16 in the main paper.

**Comment by gabrielilharco** [comment]
Hi @alirezaabdollahpour, the hyper-parameters are specified in Appendix D.2, please let us know if you have any questions

**Comment by alirezaabdollahpour** [comment]
Hi @gabrielilharco , thank you for responding. In the section you mentioned earlier, you discussed the learning rates and augmentation. What is the recommended number of epochs for fine-tuning CIFAR10 models?

**Comment by gabrielilharco** [comment]
We do 10 epochs

**Comment by alirezaabdollahpour** [comment]
Thanks a lot, @gabrielilharco.

**Comment by DongHwanJang** [comment]
I have a question about the warmup length in the paper. 

The paper states that the warmup length is 500 steps, but for CIFAR100 with a batch size of 512, one epoch only takes 98 steps to train. 
This means that the warmup phase occupies almost half of the total training time (total 10 epochs). 
Is this correct? 

It would be greatly helpful if you could clarify this point.
Thank you.

**Comment by mitchellnw** [comment]
I believe so, do you find this drops accuracy compared to e.g. 50 warmup steps?
